### ISSUE TITLE
After canceling execution due to bail (scenario failed), remaining scenarios are marked as passed anyway

### ISSUE THREAD

**Issue by Vitouuu** [webdriverio/wdio-spec-reporter#29]
TL;DR: Multiple "it" scenarios (mocha test). First one fails. Although the rest are not run, they are marked as passed:

<img width="804" alt="screen shot 2017-03-22 at 10 27 56 am" src="https://cloud.githubusercontent.com/assets/3942402/24199983/4cb34d9e-0eea-11e7-964e-762c9e0129db.png">

Steps:
1. Configure bail() = 1 so after the first fail all test execution is stopped.
2. Run multiple scenarios having Mocha as runner.
3. Make first scenario fail.
4. Observe results.

Actual result:
- All remaining scenarios, which weren't run, are marked as passed.

Expected result:
- Remaining scenarios are marked as pending or "non-executed".

Notes:
    "wdio-dot-reporter": "0.0.8",
    "wdio-mocha-framework": "0.5.9",
    "wdio-spec-reporter": "0.1.0"

**Comment by timbru31** [comment]
As far as I understand bail only exits early when a new spec would be executed, as long as it's the same spec (normally the case when you have multiple `it`s in a test).

See https://github.com/webdriverio/webdriverio/blob/8e6ea56ae14e50db34d150207246004ea73a23f8/lib/launcher.js#L213-L230

IMHO: not a reporter issue (happens on e.g. dot, too)

@christian-bromann what do you think?

**Comment by christian-bromann** [comment]
@timbru31 you are right!

There are two bail options:

- for WebdriverIO, when you don't want to launch any new tests after one failed
- and in Mocha when you don't want to continue running your test if an assertion fails

You need to add in your wdio.conf.js:
```js
  // ...
  mochaOpts: {
    // ...
    bail: true,
    //
  }
  //
```

This is not an issue.
